rosetta/views.py

```diff
@@ -155,6 +155,8 @@ def _request_request(key, default=None):
                         storage.set('rosetta_last_save_error', True)
 
             if file_change and rosetta_i18n_write:
+                pre_save.send(sender=None, language_code=rosetta_i18n_lang_code, request=request)
```

Comment:
Also, I'd rather use `send_robust` instead of plain `send` here, or include some exception handling.